### ISSUE TITLE
Double-softmax in validation loss

### ISSUE THREAD

**Issue by mdarcy220** [ariecattan/SciCo#2]
Validation loss quickly gets stuck around 1.0 and only decreases very slightly from there when I try training the model.  This seems to be due to the logits being softmaxed into probabilities before being fed into the CrossEntropyLoss [in `models/muticlass.py`](https://github.com/ariecattan/SciCo/blob/94d6d0ed4edb29d3a8fea1c4bce44720945d2c27/models/muticlass.py#L101), which treats those inputs as logits and softmaxes them again.  This bounds the minimum possible loss per example to roughly 0.74.  Training loss does not appear to have this issue, only validation and test.

**Comment by ariecattan** [comment]
Thanks @mdarcy220 for pointing this out, we've fixed the issue!
